docs/sources/configuration/integrations/integrations-next/oracledb-config.md

```diff
@@ -0,0 +1,86 @@
+---
```

Comment:
I just checked again at the docs and we don't have a doc's page for shimed v2 integrations. So we could delete this.

Reply:
Deleted 👍